### PR TITLE
messages main funcionality

### PR DESCRIPTION
**Pull request by jlguida**

### CODE REVIEW

**Comment by jlguida** [comment]
Still an error when accessing messages/ while unauthenticated

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -71,7 +74,7 @@ def set_message
     def message_params
       p = params.require(:message).permit(:receiver_id, :body)
       if current_user
-        p[:sender_id] = current_user.id
+        p[:sender_id] = session[:user_id]
```
Comment (lar-ryan-nick) [inline app/controllers/messages_controller.rb]: Why did you change this?

Comment (jlguida) [reply app/controllers/messages_controller.rb]: I was getting strange behavior and it fixed it. Are there any side effects with this? If so I can change back

Comment (lar-ryan-nick) [reply app/controllers/messages_controller.rb]: I don't know, but if it works, it works.
